pkg/config/piped.go

```diff
@@ -326,6 +329,9 @@ type PipedGit struct {
 	SSHKeyFile string `json:"sshKeyFile,omitempty"`
 	// Base64 encoded string of ssh-key.
 	SSHKeyData string `json:"sshKeyData,omitempty"`
+	// The password authentication.
+	// This will be used to clone the source code of the specified git repositories.
+	PasswordAuth PipedGitPasswordAuth `json:"passwordAuth,omitempty"`
```

Comment:
Why don't you simply use `Password`? Are there any reasons for that? Also let's make password be base64 encoded.

Reply:
Since 'Password' overlaps with the variable name, 'PersonalAccessToken' renamed to 'PasswordAuth' and 'userToken' to 'Password'.

Reply:
Sorry for confusing you. But what I meant is that we can just add password not username because username already exists in this `PipedGit` struct.